tests/h/search/query_test.py

```diff
@@ -653,6 +653,20 @@ def test_it_limits_number_of_annotation_counts_by_user_returned(self, Annotation
         assert count_pc == 2
 
 
+@pytest.fixture(params=['es1', 'es6'])
+def pyramid_request(request, pyramid_request, es_client, es6_client):
+    pyramid_request.es = es_client
+    pyramid_request.es6 = es6_client
+
+    def feature(flag):
+        if flag == 'search_es6':
+            return request.param == 'es6'
+        return True
+
+    pyramid_request.feature = mock.Mock(side_effect=feature)
```

Comment:
This can be simplified a little using `pyramid_request.feature.flags`. See comment above.